std/json.d

```diff
@@ -628,7 +655,7 @@ struct JSONValue
     /// Implicitly calls $(D toJSON) on this JSONValue.
     ///
     /// $(I options) can be used to tweak the conversion behavior.
-    string toString(in JSONOptions options = JSONOptions.none) const
+    string toString(in JSONOptions options = JSONOptions.none) const @safe
     {
         return toJSON(&this, false, options);
```

Comment:
`scope` (implied by `in`) is only implemented for trivial cases and as an optimization for delegate parameters, it cannot be relied upon for memory safety guarantees (and indeed `@safe` doesn't).